src/machines/workflow_machines.rs

```diff
@@ -459,13 +459,7 @@ impl WorkflowMachines {
     pub(crate) fn get_commands(&self) -> Vec<ProtoCommand> {
         self.commands
             .iter()
-            .filter_map(|c| {
-                if !self.machine(c.machine).is_final_state() {
-                    Some(c.command.clone())
-                } else {
-                    None
-                }
-            })
+            .filter_map(|c| (!self.machine(c.machine).is_final_state()).then(|| c.command.clone()))
```

Comment:
This one I don't totally love. I think the original is easier to parse.

Reply:
https://github.com/temporalio/sdk-core/pull/228/commits/5eb12623e7f0ead32968a9469deedcb227c397c7